cmd/influx/cli/cli.go

```diff
@@ -31,6 +31,10 @@ import (
 // ErrBlankCommand is returned when a parsed command is empty.
 var ErrBlankCommand = errors.New("empty input")
 
+// DefaultChunkSize is the default number of results to return
+// per response if chunked responses are turned on
+const DefaultChunkSize = 10000
```

Comment:
Default chunk size should just be set to the zero value. We don't want to duplicate the value from the server in the client. If the chunk size is set to zero, don't include it in the client request and that will cause the server to use the default.